tests/unit/requirements.txt

```diff
@@ -12,7 +12,7 @@ linode-python  # APIv3
 linode_api4 ; python_version > '2.6'  # APIv4
 
 # requirement for the gitlab module
-python-gitlab < 2.3.0  # version 2.3.0 makes gitlab_runner tests fail
+python-gitlab
```

Comment:
perhaps `> 2.0` or such?

Reply:
We usually do not specify explicit versions if not necessary.